.env.production

```diff
@@ -1,4 +1,2 @@
-# Authorization left empty intentionally for production
 REACT_APP_DHIS2_BASE_URL=".."
-REACT_APP_DHIS2_AUTHORIZATION=
-REACT_APP_TRACKER_CAPTURE_APP_PATH="../dhis-web-tracker-capture"
\ No newline at end of file
+REACT_APP_TRACKER_CAPTURE_APP_PATH="../dhis-web-tracker-capture"
```

Comment:
Likewise, there shouldn't be any production environment variables needed here - the platform automatically generates the baseUrl